ocp/ansible/roles/zoscb/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+- include_vars: zoscb.yml
+  tags: zoscb
+# Create namespace for cloudbroker
+# tasks file for zoscb
+- name: "Create project with Native YAML API"
+  kubernetes.core.k8s:
+    state: present
+    src: "files/project.yaml"
+    wait: true
+    wait_sleep: 50
+  tags: zoscb
+
+- name: "Create PVC with Native YAML API"
+  kubernetes.core.k8s:
+    state: present
+    namespace: "ibm-zmodstack-cloudbroker"
+    src: "files/persistent-volume-claim.yaml"
+    wait: true
+    wait_sleep: 50
+  tags: zoscb
+
+- name: "Create Operator Group with Native YAML API"
+  kubernetes.core.k8s:
+    state: present
+    namespace: "ibm-zmodstack-cloudbroker"
+    src: "files/operator-group.yaml"
+    wait: true
+    wait_sleep: 50
+  tags: zoscb
+
+- name: "Create brokerInstance Subscription with Native YAML API"
+  kubernetes.core.k8s:
+    state: present
+    namespace: "ibm-zmodstack-cloudbroker"
+    src: "files/broker-subscription.yaml"
+    wait: true
+    wait_sleep: 50
+  tags: zoscb
+
+- name: "Retrieve broker subscription info"
+  kubernetes.core.k8s_info:
+    kind: Subscription
+    namespace: "ibm-zmodstack-cloudbroker"
+    name: "{{ broker_subscription.name }}"
+    api_version: "{{ broker_subscription.apiVersion }}"
+  register: broker_subscription_info
+  until:
+    - "broker_subscription_info.resources is defined and broker_subscription_info.resources | length > 0"
+    - "'state' in broker_subscription_info.resources[0].status and broker_subscription_info.resources[0].status.state == 'AtLatestKnown'"
+  retries: 30
+  delay: 10
```

Comment:
Provisioning seems to get stuck on this condition

Reply:
The namespace parameter was missing while re-doing the kubeconfig logic 
I have updated this in the following commit 
Thanks